<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#titleLabel {
background: white;
color: blue;
font-size: 20px;
border: none;
border-bottom:  1px solid black;
padding: 5px;
}

#mainFrame {
border: none;
background: white;
}

#loginFrame {
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ddf, stop: 1 #aaf);
border: 1px solid gray;
padding: 10px;
border-radius: 25px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="mainWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="chatPage">
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="2">
            <widget class="QWidget" name="widget" native="true">
             <widget class="QPushButton" name="sendButton">
              <property name="geometry">
               <rect>
                <x>330</x>
                <y>330</y>
                <width>61</width>
                <height>21</height>
               </rect>
              </property>
              <property name="text">
               <string>Send</string>
              </property>
              <property name="shortcut">
               <string>Return</string>
              </property>
             </widget>
             <widget class="QListWidget" name="usersListWidget">
              <property name="geometry">
               <rect>
                <x>330</x>
                <y>30</y>
                <width>131</width>
                <height>161</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
             </widget>
             <widget class="QLabel" name="label_3">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>81</width>
                <height>16</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Chat room</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_4">
              <property name="geometry">
               <rect>
                <x>330</x>
                <y>10</y>
                <width>111</width>
                <height>16</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Who is online</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_5">
              <property name="geometry">
               <rect>
                <x>330</x>
                <y>200</y>
                <width>47</width>
                <height>13</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Smiles</string>
              </property>
             </widget>
             <widget class="QTextEdit" name="textEdit">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>30</y>
                <width>311</width>
                <height>281</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
             <widget class="QWidget" name="smileswidget" native="true">
              <property name="geometry">
               <rect>
                <x>330</x>
                <y>220</y>
                <width>131</width>
                <height>91</height>
               </rect>
              </property>
              <widget class="QPushButton" name="smile0">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>10</y>
                 <width>20</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="images.qrc">
                 <normaloff>:/images/Smiles/0.png</normaloff>:/images/Smiles/0.png</iconset>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup_2</string>
               </attribute>
              </widget>
              <widget class="QPushButton" name="smile1">
               <property name="geometry">
                <rect>
                 <x>40</x>
                 <y>10</y>
                 <width>21</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="images.qrc">
                 <normaloff>:/images/Smiles/1.png</normaloff>:/images/Smiles/1.png</iconset>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup_2</string>
               </attribute>
              </widget>
              <widget class="QPushButton" name="smile2">
               <property name="geometry">
                <rect>
                 <x>70</x>
                 <y>10</y>
                 <width>21</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="images.qrc">
                 <normaloff>:/images/Smiles/2.png</normaloff>:/images/Smiles/2.png</iconset>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup_2</string>
               </attribute>
              </widget>
              <widget class="QPushButton" name="smile3">
               <property name="geometry">
                <rect>
                 <x>100</x>
                 <y>10</y>
                 <width>21</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="images.qrc">
                 <normaloff>:/images/Smiles/3.png</normaloff>:/images/Smiles/3.png</iconset>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup_2</string>
               </attribute>
              </widget>
              <widget class="QPushButton" name="smile4">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>40</y>
                 <width>21</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="images.qrc">
                 <normaloff>:/images/Smiles/4.png</normaloff>:/images/Smiles/4.png</iconset>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup_2</string>
               </attribute>
              </widget>
              <widget class="QPushButton" name="smile5">
               <property name="geometry">
                <rect>
                 <x>40</x>
                 <y>40</y>
                 <width>21</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="images.qrc">
                 <normaloff>:/images/Smiles/5.png</normaloff>:/images/Smiles/5.png</iconset>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup_2</string>
               </attribute>
              </widget>
              <widget class="QPushButton" name="smile6">
               <property name="geometry">
                <rect>
                 <x>70</x>
                 <y>40</y>
                 <width>21</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="images.qrc">
                 <normaloff>:/images/Smiles/6.png</normaloff>:/images/Smiles/6.png</iconset>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup_2</string>
               </attribute>
              </widget>
              <widget class="QPushButton" name="smile7">
               <property name="geometry">
                <rect>
                 <x>100</x>
                 <y>40</y>
                 <width>21</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="images.qrc">
                 <normaloff>:/images/Smiles/7.png</normaloff>:/images/Smiles/7.png</iconset>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup_2</string>
               </attribute>
              </widget>
              <widget class="QPushButton" name="smile8">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>70</y>
                 <width>21</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="images.qrc">
                 <normaloff>:/images/Smiles/8.png</normaloff>:/images/Smiles/8.png</iconset>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup_2</string>
               </attribute>
              </widget>
              <widget class="QPushButton" name="smile10">
               <property name="geometry">
                <rect>
                 <x>70</x>
                 <y>70</y>
                 <width>21</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="images.qrc">
                 <normaloff>:/images/Smiles/10.png</normaloff>:/images/Smiles/10.png</iconset>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup_2</string>
               </attribute>
              </widget>
              <widget class="QPushButton" name="smile11">
               <property name="geometry">
                <rect>
                 <x>100</x>
                 <y>70</y>
                 <width>21</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="images.qrc">
                 <normaloff>:/images/Smiles/11.png</normaloff>:/images/Smiles/11.png</iconset>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup_2</string>
               </attribute>
              </widget>
              <widget class="QPushButton" name="smile9">
               <property name="geometry">
                <rect>
                 <x>40</x>
                 <y>70</y>
                 <width>20</width>
                 <height>20</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="images.qrc">
                 <normaloff>:/images/Smiles/9.png</normaloff>:/images/Smiles/9.png</iconset>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup_2</string>
               </attribute>
              </widget>
             </widget>
             <widget class="QLineEdit" name="messageLineEdit">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>330</y>
                <width>311</width>
                <height>21</height>
               </rect>
              </property>
             </widget>
             <widget class="QPushButton" name="fileButton">
              <property name="geometry">
               <rect>
                <x>400</x>
                <y>330</y>
                <width>61</width>
                <height>21</height>
               </rect>
              </property>
              <property name="text">
               <string>Send File</string>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="loginPage">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="2" column="0">
            <widget class="QWidget" name="widget_2" native="true">
             <widget class="QPushButton" name="randButton">
              <property name="geometry">
               <rect>
                <x>320</x>
                <y>90</y>
                <width>81</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>Random</string>
              </property>
             </widget>
             <widget class="QPushButton" name="loginButton">
              <property name="geometry">
               <rect>
                <x>140</x>
                <y>340</y>
                <width>261</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>Login</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="serverLineEdit">
              <property name="geometry">
               <rect>
                <x>140</x>
                <y>60</y>
                <width>261</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text">
               <string>127.0.0.1</string>
              </property>
             </widget>
             <widget class="QLabel" name="label">
              <property name="geometry">
               <rect>
                <x>60</x>
                <y>60</y>
                <width>62</width>
                <height>21</height>
               </rect>
              </property>
              <property name="text">
               <string>Server Name</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
             <widget class="QLabel" name="label_2">
              <property name="geometry">
               <rect>
                <x>70</x>
                <y>90</y>
                <width>52</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text">
               <string>User Name</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
             <widget class="QLineEdit" name="userLineEdit">
              <property name="geometry">
               <rect>
                <x>140</x>
                <y>90</y>
                <width>171</width>
                <height>20</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="label_6">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>130</y>
                <width>91</width>
                <height>31</height>
               </rect>
              </property>
              <property name="text">
               <string>Select your 
profile picture</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
             <widget class="QWidget" name="verticalLayoutWidget">
              <property name="geometry">
               <rect>
                <x>320</x>
                <y>130</y>
                <width>84</width>
                <height>212</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QRadioButton" name="radioButton_1">
                 <property name="text">
                  <string>Angel</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_2">
                 <property name="text">
                  <string>Engineer</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_3">
                 <property name="text">
                  <string>King</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_4">
                 <property name="text">
                  <string>Lady</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_5">
                 <property name="text">
                  <string>Mexican</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_6">
                 <property name="text">
                  <string>Policeman</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_7">
                 <property name="text">
                  <string>Professor</string>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_8">
                 <property name="text">
                  <string>Superman</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_9">
                 <property name="text">
                  <string>Uncle Sam</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroup</string>
                 </attribute>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QLabel" name="label_7">
              <property name="geometry">
               <rect>
                <x>50</x>
                <y>340</y>
                <width>71</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>Click to log in</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
             <widget class="QLabel" name="avatarslabel">
              <property name="geometry">
               <rect>
                <x>140</x>
                <y>130</y>
                <width>171</width>
                <height>201</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
